src/hooks/screen/useChatScreenHook.ts

```diff
@@ -14,7 +14,6 @@ import UserSchema from '../../database/schema/UserSchema';
 import currentChatScreenAtom from '../../atom/currentChatScreenAtom';
 import useAnalyticUtilsHook from '../../libraries/analytics/useAnalyticUtilsHook';
 import useChatUtilsHook from '../core/chat/useChatUtilsHook';
-import useDatabaseQueueHook from '../core/queue/useDatabaseQueueHook';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import useUserAuthHook from '../core/auth/useUserAuthHook';
 import ImageUtils, {UploadOptions} from '../../utils/image';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useDatabaseQueueHook` has been removed. This could potentially impact the functionality if it was used in the code. However, based on the changes made in this PR, it seems like the usage of `useDatabaseQueueHook` has been replaced with direct database operations and a new function `handleChatData`. Therefore, this change should not cause any issues as long as all previous functionalities are covered.
```diff
- import useDatabaseQueueHook from '../core/queue/useDatabaseQueueHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->